targets/targets.json

```diff
@@ -345,7 +343,19 @@
         },
         "overrides": {
             "network-default-interface-type": "ETHERNET"
-        }
+        },
+        "supported_c_libs": {
+            "arm": [
+                "std", "small"
+            ],
+            "gcc_arm": [
+                "std", "small"
+            ]
+        },
+        "c_lib": "small",
```

Comment:
This should only be here if it is a former Mbed 2 target (which I do not it is). @rajkan01 Can you confirm?

Reply:
Yes target config "c_lib": _"small"_ should be there for the Mbed 2 target